test/bats/tests/e2e_provider/e2e_provider_v1alpha1_secretproviderclass_ns.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: e2e-provider-sync
+  namespace: default
+spec:
+  provider: invalidprovider
+  secretObjects:
+  - secretName: foosecret
+    type: Opaque
+    data: 
+    - objectName: $SECRET_NAME
+      key: username
+  parameters:
+    objects: |
+      array:
+        - |
+          objectName: $SECRET_NAME
+          objectType: secret
+          objectVersion: $SECRET_VERSION
+        - |
+          objectName: $KEY_NAME
+          objectType: key
+          objectVersion: $KEY_VERSION
+---
+apiVersion: secrets-store.csi.x-k8s.io/v1alpha1
+kind: SecretProviderClass
+metadata:
+  name: e2e-provider-sync
+  namespace: test-ns
+spec:
+  provider: e2e-provider
+  secretObjects:
+  - secretName: foosecret
+    type: Opaque
+    data: 
+    - objectName: $SECRET_NAME
+      key: username
+  parameters:
+    objects: |
+      array:
+        - |
+          objectName: $SECRET_NAME
+          objectType: secret
```

Comment:
`objectType` has no significance in the mock provider, so we should remove it in all the yamls

Reply:
removed